sdcm/cluster.py

```diff
@@ -311,9 +311,15 @@ def __init__(self, name, parent_cluster, ssh_login_info=None, base_logdir=None,
 
     def save_cqlsh_output_in_file(self, cmd: str, log_file: str):
         self.log.info("Save command '%s' output in the file. Node %s", cmd, self.name)
-        result_tables = self.run_cqlsh(cmd, split=True)
-        for line in result_tables:
-            self.remoter.run(f"echo '{line}' >> {log_file}")
+        file_path = self.remoter.sudo(f"touch {log_file};realpath {log_file}",
+                                      verbose=True, ignore_status=True).stdout.strip()
+        if not (result_tables := self.run_cqlsh(cmd).stdout):
+            return
+
+        self.log.debug("Schema file path: %s", file_path)
+        with remote_file(remoter=self.remoter, remote_path=file_path, sudo=True) as fobj:
```

Comment:
let's save this into sct for folder, and not into the node

it would be hard to look it up, now that we won't know in which node it was collected